snapcraft.yaml

```
@@ -78,3 +78,4 @@ apps:
 architectures:
   - build-on: i386
   - build-on: amd64
+  - build-on: arm64
```

Please also add `armhf` as starship also distributes binaries for that architecture.
```suggestion
  - build-on: arm64
  - build-on: armhf
```


I don't know if they build right now but you could also add `s390x` and `ppc64el`, but with `build-error: ignore`?
I think that's the syntax:
```suggestion
  - build-on: arm64
  - build-on: armhf
  - build-on: s390x
    build-error: ignore
  - build-on: ppc64el
    build-error: ignore
```

---

Tried them all, this is the results:

```text
	arch=arm64	state=Successfully built
	arch=armhf	state=Successfully built
	arch=i386	state=Successfully built
	arch=amd64	state=Successfully built
	arch=ppc64el	state=Successfully built
	arch=s390x	state=Failed to build
```

Will update accordingly 🚀